src/devices/controller/controller.cpp

```diff
@@ -89,36 +114,36 @@ std::array<uint8_t, 6> palettes = {
 void RunEffectMode() {
   const uint8_t num_unique_effects = led_manager.GetNumUniqueEffects();
   const uint8_t num_palettes = Effect::palettes().size();
-  if (left_buttons.Button1Pressed()) {
+  if (left_buttons[0].Rose()) {
```

Comment:
Rose as in the button is depressed and shorts? Or rose as in the physical button is lifted?

Reply:
Rose as in the logical signal rose - since this uses `ForInvertedDigitalRead`, `Rose` means the button was pressed.